### PR TITLE
Remove sourcemap validation

### PR DESCRIPTION
**Pull request by lNikolayl**
Comment sourcemap validation due to new behavior in babel sourcemap generation. Probably babel is stable and we don't need validation as it is.

### CODE REVIEW

File: lib/source-map.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ const crypto = require('crypto');
 const chalk = require('chalk');
 const EOL = require('os').EOL;
 const endsWith = require('./endsWith');
-const validator = require('sourcemap-validator');
+// const validator = require('sourcemap-validator');
```
Comment (andreyfel) [inline lib/source-map.js]: Just remove the code, do not comment it out. And please remove the dep from the dependencies.

File: lib/source-map.js [hunk 2]
```diff
@@ -114,20 +114,23 @@ class SourceMap {
 
     let valid = true;
```
Comment (andreyfel) [inline lib/source-map.js]: This variable is not needed once you remove the validation.

File: lib/source-map.js [hunk 3]
```diff
@@ -114,20 +114,23 @@ class SourceMap {
...
+    //   } catch (e) {
+    //     logger.error(' invalid sourcemap for: %s', filename);
+    //     if (typeof e === 'object' && e !== null) {
+    //       logger.error('   error: ', e.message);
+    //     }
+
+    //     // print
+    //     valid = false;
+    //   }
+    // }
```
Comment (andreyfel) [inline lib/source-map.js]: Remove the code instead of commenting.

**Review by andreyfel — CHANGES_REQUESTED** [review]

File: lib/source-map.js (hunk 3 above)
Comment (lNikolayl) [reply lib/source-map.js]: Done

File: lib/source-map.js (hunk 2 above)
Comment (lNikolayl) [reply lib/source-map.js]: Done

File: lib/source-map.js (hunk 1 above)
Comment (lNikolayl) [reply lib/source-map.js]: Done

**Review by andreyfel — APPROVED** [review]

**Pull request merged**